src/business-logic/services/test/throttled-rundown-service.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { ThrottledRundownService } from '../throttled-rundown-service'
+import { anything, instance, mock, verify } from '@typestrong/ts-mockito'
+import { ThrottledRundownException } from '../../../model/exceptions/throttled-rundown-exception'
+import { RundownService } from '../interfaces/rundown-service'
+
+describe(ThrottledRundownService.name, () => {
+  beforeEach(() => jest.useFakeTimers())
+  afterEach(() => jest.useRealTimers())
+  describe('when two operations are performed within the throttled interval', () => {
```

Comment:
It would be nice with a space between the `afterEach` and the `describe`.

Reply:
Added